plugins.go

```diff
@@ -116,16 +107,98 @@ func findAndLoadPlugins(o *Options) error {
 	return nil
 }
 
+func (o *Options) LoadMultiPlugins(found map[string]string) error {
```

Comment:
I think that the methods matching `Load.+Plugins?` here should not be exported. Is there a reason for exporting them, or a use case for applications that import skipper? I see that they are already exported in the master, but maybe it's not late to change this, if there is no reasonable use case for it. What may it break if unexporting them all?